examples/sagemaker/endpoint.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: sagemaker.aws.upbound.io/v1beta1
+kind: Endpoint
+metadata:
+  annotations:
+    meta.upbound.io/example-id: sagemaker/v1beta1/endpoint
+    upjet.upbound.io/manual-intervention: "This resource is skipping because the EndpointConfiguration resource on which this resource depends was skipped."
+  labels:
+    testing.upbound.io/example-name: e
+  name: e
+spec:
+  forProvider:
+    endpointConfigNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: ec
+    region: us-east-1
+    tags:
+      Name: foo
+---
+apiVersion: sagemaker.aws.upbound.io/v1beta1
+kind: EndpointConfiguration
+metadata:
+  annotations:
+    meta.upbound.io/example-id: sagemaker/v1beta1/endpoint
+    upjet.upbound.io/manual-intervention: "It requires an image which can be create/deleted only using AWS CLI"
+  labels:
+    testing.upbound.io/example-name: ec
+  name: ec
+spec:
+  forProvider:
+    productionVariants:
+    - initialInstanceCount: 1
+      instanceType: ml.t2.medium
+      modelNameSelector:
+        matchLabels:
+          testing.upbound.io/example-name: example
+      variantName: variant-1
+    region: us-east-1
+    tags:
+      Name: foo
+---
+apiVersion: sagemaker.aws.upbound.io/v1beta1
+kind: Model
+metadata:
+  annotations:
+    meta.upbound.io/example-id: sagemaker/v1beta1/endpoint
+    upjet.upbound.io/manual-intervention: "It requires an image which can be create/deleted only using AWS CLI"
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    executionRoleArnSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    primaryContainer:
+      - image: 683313688378.dkr.ecr.us-east-1.amazonaws.com/sagemaker-scikit-learn:0.23-1-cpu-py3
```

Comment:
```suggestion
      - image: ${data.aws_account_id}.dkr.ecr.us-east-1.amazonaws.com/sagemaker-scikit-learn:latest
```

Reply:
Totally makes sense. I wasn't overly concerned just for the fact that this is a public image but I agree with your comment here.